packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -19,6 +17,11 @@ export interface ISwatchColorPickerProps {
   id?: string;
 
   /**
+   * Additional class name to provide on the root element
+   */
+  className?: string;
```

Comment:
That would be a separate PR. Adding this overriding className is a piece of feedback from David